### ISSUE TITLE
What's the difference between s and inv_s in function render_core?

### ISSUE THREAD

**Issue by lambdald** [yiqun-wang/HFS#6]
The names and formulas of these two variables confused me.
`
s = deviation_network(torch.zeros([1])).clip(1e-6, 1e3).detach()
`
`
inv_s = deviation_network(torch.zeros([1, 3]))[:, :1].clip(1e-6, 1e6)
`

**Comment by yiqun-wang** [comment]
Hi, thanks for the reminder. The rules of the names need to be further improved, there is no special meaning at present.
